### ISSUE TITLE
CLI not using args/options

### ISSUE THREAD

**Issue by Blankeos** [captbaritone/grats#12]
I read this: https://grats.capt.dev/docs/usage/cli
- `grats -V` doesn't show the version.
- `grats --output=./schema.graphql` does not do anything.

![grats cli issue](https://github.com/captbaritone/grats/assets/38070918/7a846da1-a4ee-4f83-9418-48284883303e)

My TsConfig:
```json
{
  "compilerOptions": {
    "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
    "module": "commonjs",                                /* Specify what module code is generated. */
    "esModuleInterop": true,                             /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */
    "forceConsistentCasingInFileNames": true,            /* Ensure that casing is correct in imports. */
    "strict": true,                                      /* Enable all strict type-checking options. */
    "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
  }
}
```

**Comment by captbaritone** [comment]
Ah, this capability has been added but has not yet made it into a release. Version 0.0.2 does not have it yet.

**Comment by captbaritone** [comment]
I've just released version 0.0.3 which should remedy this.
